frontend/src/locales/en.json

```
@@ -2120,7 +2283,37 @@
     "reviewApiKey": "Review API Keys",
     "reviewApiKeyPlaceholder": "One key per line (sk-...), multiple allowed",
     "reviewApiKeyConfigured": "{{n}} key(s) configured; leave blank to keep",
-    "reviewApiKeyHint": "One key per line. Add multiple keys to share the Moderations TPM quota; on rate limits it automatically fails over to the next key.",
+    "reviewApiKeyHint": "One key per line. Multiple keys are round-robin balanced and failed over on rate limits, invalid keys, or server errors.",
+    "reviewRequestMode": "Request Adapter",
+    "reviewModeChat": "Chat Completions compatible",
+    "reviewModeModerations": "Moderations (legacy compatible)",
+    "reviewScope": "Review Scope",
```

Biome (2.5.5)

[error] The key reviewScope was already declared.

(lint/suspicious/noDuplicateObjectKeys)